framework/test/unit/modules/pos/module.spec.ts

```diff
@@ -1139,10 +1140,58 @@ describe('PoS module', () => {
 
```

Comment:
Test of line 1122: I don't fully understand what we want to test here and if it is properly tested.

Reply:
description was wrong, but it was testing when snapshot validators length is small, it should select remaining validators from init validators